web/packages/teleterm/src/services/tshd/interceptors.ts

```diff
@@ -35,6 +35,13 @@ const SENSITIVE_PROPERTIES = [
   'commands',
 ];
 
+// Skips logging requests/responses for given methods.
+// Only errors are logged.
+const OMITTED_METHODS = new Set([
```

Comment:
Oh wait so these are actually called "methods", I was wrong. https://grpc.io/docs/what-is-grpc/core-concepts/#service-definition